app/assets/stylesheets/components/common/_global_nav.scss

```diff
@@ -135,6 +135,18 @@ $global-nav-transition-duration: 300ms;
   }
 }
 
+.touch {
+  .global-nav__clump {
+    &:hover {
+      background: transparent;
+    }
+
+    &.is-active {
+      background: $color-white;
+    }
+  }
+}
```

Comment:
Hmm, not sure, I was kind of anticipating that the .touch class may get used more widely (though admittedly I don't know what for right now). Was actually considering moving it to the end of the file with the flexbox-specific stuff. Nonetheless I saw it as a more generic property.